### PR TITLE
#170947533, #170947534, #170947539 set-up postgres, sequelize and dotenv

### PR DESCRIPTION
**Pull request by bbaime98**
#### What does this PR do?
- set-up database with sequelize   and secure their credentials with dotenv
#### Description of Task to be completed?
- install pg and pg-hstore dependencies
- install sequelize, sequelize-cli, and dotenv dependencies
- add  sequelizelrc file
- add config file  with environment variables
#### How should this be manually tested?
- clone repo
- add your postgres credentials in `.env` file
- run `sequelize db:migrate`
- add dummy user in seeder file
- run `sequelize db:seed:all`
#### What are the relevant pivotal tracker stories?
- [#170947533](https://www.pivotaltracker.com/story/show/170947533)
- [#170947539](https://www.pivotaltracker.com/story/show/170947539)
- [#170947534](https://www.pivotaltracker.com/story/show/170947534)
#### Questions:
- N/A

### CODE REVIEW

**Review by Ntare22 — COMMENTED** [review]
I do not see the migrations, were the files created in the database?

**Comment by bbaime98** [comment]
> I do not see the migrations, were the files created in the database?

after setting up both Postgres and sequelize, I used dummy data to test if it works by creating seeders and migrating them. Ultimately I deleted them.

**Review by Ntare22 — DISMISSED** [review]

**Review by Baraka-Mugisha — DISMISSED** [review]

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports =  {
+  'config': path.resolve('src','config', 'config.js'),
+  'models-path': path.resolve('src','models'),
+  'seeders-path': path.resolve('src','seeders'),
+  'migrations-path': path.resolve('src','migrations')
+}
```
Comment (Mnickii) [inline .sequelizerc]: Add a new line for consistency with other files. You can configure your editor to automatically do this.

**Review by Mnickii — CHANGES_REQUESTED** [review]
Also, let's include the migrations so that we can track any changes, and seeders, and a script to migrate on package.json

**Review by Ntare22 — DISMISSED** [review]
Finally man!!

File: .hound.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
 eslint:
   enabled: true
-  config_file: .eslintrc
-fail_on_violations: true
```
Comment (Mnickii) [inline .hound.yml:4]: return this, let it look like this 
```
eslint:
  enabled: true
  config_file: .eslintrc.json
  ignore_file: .eslintignore

fail_on_violations: true
```

**Review by Mnickii — CHANGES_REQUESTED** [review]

**Review by StevenDerrick — APPROVED** [review]

File: .hound.yml (hunk 2 above)
Comment (bbaime98) [reply .hound.yml:4]: It is fixed.

**Review by Baraka-Mugisha — APPROVED** [review]

**Review by Mnickii — APPROVED** [review]

**Pull request merged**